server/requirements.txt

```diff
@@ -7,3 +7,5 @@ oauth2client==4.1.3
 -e git+git://github.com/superdesk/superdesk-core.git@develop#egg=Superdesk-Core
 -e git+git://github.com/superdesk/superdesk-analytics.git@master#egg=superdesk-analytics
 -e git+git://github.com/superdesk/superdesk-planning.git@master#egg=superdesk-planning
+
+flask-oidc==1.4.0
```

Comment:
When we put `flask-oidc` at the begin of file, I conflict with some library in Superdesk-Core and cannot install, So we add it in the end of file.